### ISSUE TITLE
Document the error that comes from using default notary list docker.io/bla/

### ISSUE THREAD

**Issue by diogomonica** [docker/notary#459]
Right now the default remote notary server for notary client is `notary-server:4443`. This means that a fresh install of notary will have this behavior:

```
➜   notary list docker.io/diogomonica/alpine
* fatal: notary-server:4443 does not have trust data for docker.io/diogomonica/alpine
```

We eventually want to point the default server to thenotary.io, but for now we should at least document that people that get this error, should change their `~/.notary/config.json` to have:

```
{
	"remote_server": {
		"url": "https://notary.docker.io"
	}
}
```

**Comment by riyazdf** [comment]
Just noting that this will be resolved after merging #293

**Comment by endophage** [comment]
#293 is superseded by #603
